tests/integration/templates/events_rule_without_targets.yaml

```diff
@@ -0,0 +1,6 @@
+Resources:
+  TestRule99A50909:
+    Type: AWS::Events::Rule
+    Properties:
+      Name: {{ event_rule_name }}
```

Comment:
Please add this as a normal CloudFormation Parameter for the template instead.

i.e. something like
```
Parameters:
    EventRuleName:
        Type: String
```

Trying to enforce this more now :+1: